### PR TITLE
🚸(edxapp) provision demo test and staff users

### PR DESCRIPTION
**Pull request by rmoch**

### CODE REVIEW

File: group_vars/all/main.yml [hunk 1]
```diff
@@ -114,3 +114,34 @@ openshift_volumes:
...
+    username: staff
+    hashed_password: "{{ demo_hashed_password }}"
+    is_staff: true
+    is_superuser: false
+  - email: 'admin@example.com'
+    username: admin
+    hashed_password: "{{ demo_hashed_password }}"
+    is_staff: true
+    is_superuser: true
+demo_test_staff_users: "{{ demo_test_users }} + {{ demo_staff_users }}"
```
Comment (sampaccoud) [inline group_vars/all/main.yml]: A better name may be `demo_all_users` ??

File: templates/openshift/edxapp/job/import_demo_course.yml.j2 [hunk 2]
```diff
@@ -29,7 +29,17 @@ spec:
                 cd /tmp &&
                 tar xzf /tmp/demo_course.tgz &&
                 cd - &&
-                python manage.py cms import /edx/app/edxapp/data /tmp/edx-demo-course*
+                python manage.py cms import /edx/app/edxapp/data /tmp/edx-demo-course* &&
+                {% for user in demo_test_staff_users %}
+                python manage.py lms manage_user {{ user.username}} {{ user.email }} --initial-password-hash {{ user.hashed_password | quote }} {% if user.is_staff %}--staff{% endif %} {% if user.is_superuser %}--superuser{% endif %}
```
Comment (sampaccoud) [inline templates/openshift/edxapp/job/import_demo_course.yml.j2]: add indentation as discussed with @jmaupetit ?

File: group_vars/all/main.yml [hunk 3]
```diff
@@ -114,3 +114,34 @@ openshift_volumes:
   - openshift/edxapp/pvc/static.yml
   - openshift/richie/pvc/media.yml
   - openshift/richie/pvc/static.yml
+
+  # default users for development environment
```
Comment (sampaccoud) [inline group_vars/all/main.yml]: it should be `development` AND `feature` environments

File: group_vars/all/main.yml (hunk 1 above)
Comment (rmoch) [reply group_vars/all/main.yml]: I was reusing edX variable names

**Comment by jmaupetit** [comment]
As discussed with @sampaccoud, one way to improve handling course import and avoid polluting the main configuration is:

* create a `groups_vars/all/fixtures.yml` that contains users data,
* add a `load_fixtures.yml` playbook that loads these variable definitions and create _ad hoc_ job objects.

File: README.md [hunk 4]
```diff
@@ -79,8 +79,27 @@ $ bin/init
...
 Tadaaa! Arnold has created a new OpenShift project called `patient0-development`
-with a collection of services up and running. You can change the customer /
-environment to deploy by overriding default environment variables:
+with a collection of services up and running.
+
+Note that the `edxapp-dbmigrate-init` job may take some time. When it has
+completed successfully, you can create a demo course and some users by running:
+
+```bash
+$ bin/run load_fixtures.yml
```
Comment (jmaupetit) [inline README.md]:
```
bin/ansible-playbook load_fixtures.yml
```

**Review by jmaupetit — APPROVED** [review]
:tada:

**Pull request merged**